### DISCUSSION TITLE
在哪里提交 js脚本 ，在这里吗？

### DISCUSSION THREAD

**Discussion by docker-master** [hslr-s/sun-panel#232] [Q&A]
[mouse.zip](https://github.com/user-attachments/files/16837789/mouse.zip)
自定义 页脚 <script src="/custom/mouse.js"></script>
这是鼠标 烟花特效

**Comment by hslr-s** [comment]
已经添加到插件项目中[插件项目](https://github.com/hslr-s/sun-panel-js-plugins/tree/master/clickFireworksEffects)
